lib/ruby_terraform/commands/apply.rb

```diff
@@ -7,7 +7,7 @@ class Apply < Base
       def configure_command(builder, opts)
         directory = opts[:directory]
         vars = opts[:vars] || {}
-        var_file = opts[:var_file]
+        var_file = opts[:var_file].kind_of?(String) ? opts[:var_file].lines : opts[:var_file] || []
```

Comment:
This gets duplicated across a number of commands - would it be possible to extract it somewhere?

Not sure of a good place for it, potentially in the base command for the time being, or in a utility module.